stylesheets/manifest.css

```diff
@@ -1967,9 +1965,6 @@ li.entry .error-icon-container {
     .android-dark .search::selection {
       background: white;
       color: #454545; }
-    .android-dark .search::-webkit-search-cancel-button {
-      background: url("/images/x_white.svg") no-repeat center;
-      background-size: cover; }
```

Comment:
The whole file, not just those lines. Sorry, first time I use that review functionality.